edx_proctoring/views.py

```diff
@@ -474,44 +474,57 @@ def put(self, request, attempt_id):
             )
             raise StudentExamAttemptDoesNotExistsException(err_msg)
 
-        # make sure the the attempt belongs to the calling user_id
-        if attempt['user']['id'] != request.user.id:
-            err_msg = (
-                u'Attempted to access attempt_id {attempt_id} but '
-                u'does not have access to it.'.format(
-                    attempt_id=attempt_id
-                )
+        course_id = attempt['proctored_exam']['id']
+        user_id = request.user.id
+        action = request.data.get('action')
+
+        err_msg = (
+            u'Attempted to access attempt_id {attempt_id} but '
```

Comment:
#nit `u` prefix could be omitted since we on py3 (here and below). Sorry for the annoying messages.

Reply:
No worries. Thank you for pointing it out.